src/Author.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Objects;
+
+public class Author {
+    private String authorName;
```

Comment:
Свойство "имя автора" класса "автор" это масло маслянное. У автора есть свойство "имя"
```suggestion
    private String name;
```